src/patchkernel/cell.cpp

```diff
@@ -634,13 +634,16 @@ long * Cell::getInterfaces(int face)
 */
 int Cell::findInterface(int face, int interface)
 {
-	long *faceInterfaces = getInterfaces(face);
-	int nFaceInterfaces = getInterfaceCount(face);
-	for (int i = 0; i < nFaceInterfaces; i++) {
-		if (faceInterfaces[i] == interface) {
-			return i;
-		}
-	}
+    int nFaceInterfaces = getInterfaceCount(face);
+    if (nFaceInterfaces > 0) {
```

Comment:
What clang doesn't know is that in all cases where faceInterfaces is null, nFaceInterfaces is equal to zero. Would it be possible to silence the warning issued by clang using an assert? Something like "assert(nFaceInterfaces == 0 || faceInterfaces);"

Reply:
https://github.com/optimad/bitpit/pull/304/files

Reply:
^^ PR above fixes the root cause behind these undefined behaviours